install_config/configuring_local.adoc

```diff
@@ -0,0 +1,180 @@
+
+The local volume provisioner allows using local storage within {product-title} and supports:
+
+* Volumes, and
+* Persistent Volumes
+
+[NOTE]
+====
+Local volume is an alpha feature and may change in a future release of {product-title}.
+====
+
+[[local-volume-enabling-local-volumes]]
+=== Enable Local Volumes
+Enable the `PersistentLocalVolumes` feature gate on all masters and nodes.
+
+. Edit or create the master configuration file on all masters (*_/etc/origin/master/master-config.yaml_* by default) and add  `PersistentLocalVolumes=true` under the `apiServerArguments` and `controllerArguments` sections:
++
+[source, yaml]
+----
+apiServerArguments:
+   feature-gates:
+   - PersistentLocalVolumes=true
+...
+
+ controllerArguments:
+   feature-gates:
+   - PersistentLocalVolumes=true
+...
+----
+
+. On all nodes, edit or create the node configuration file (*_/etc/origin/node/node-config.yaml_* by default)  and add `PersistentLocalVolumes=true` fetaure gate under `kubeletArguments`.
++
+[source, yaml]
+----
+kubeletArguments:
+   feature-gates:
+     - PersistentLocalVolumes=true
+----
+
+[[local-volume-mounting-local-volumes]]
+=== Mount Local Volumes
+All local volumes must be manually mounted before they can be consumed by {product-title} as persistent volumes.
+
+All volumes must be mounted into the
+*_/mnt/local-storage/<storage-class-name>/<volume>_* path. The administrators are required to create the local devices as needed (by using any method such as
+a disk partition or an LVM), create suitable filesystems on these devices, and mount them by a script or by `/etc/fstab` entries.
+
+.Example `/etc/fstab` entries:
+
+[source]
+----
+# device name   # mount point                  # FS    # options # extra
+/dev/sdb1       /mount/local-storage/ssd/disk1 ext4     defaults 1 2
+/dev/sdb2       /mount/local-storage/ssd/disk2 ext4     defaults 1 2
+/dev/sdb3       /mount/local-storage/ssd/disk3 ext4     defaults 1 2
+/dev/sdc1       /mount/local-storage/hdd/disk1 ext4     defaults 1 2
+/dev/sdc2       /mount/local-storage/hdd/disk2 ext4     defaults 1 2
+----
+
+[[local-volume-configure-local-provisioner]]
+=== Configure Local Provisioner
+OpenShift depends on an external provisioner to create persistent volumes for local devices and to clean them up when they are not needed (to enable reuse).
+
+[NOTE]
+====
+. The local volume provisioner is different from most provisioners and does not support dynamic provisioning.
+. Local volume provisioner requires that the administrators preconfigure the local volumes on each node and mount them under discovery directories. The provisioner then manages the volumes by creating and cleaning up PersistentVolumes for each volume.
+====
+
```

Comment:
Should these be bullets * and not numbered . ?
S/Local volume provisioner/The local volume provisioner